doc/api/cli.md

```diff
@@ -443,6 +443,63 @@ Affects the default output directory of:
 * [`--heap-prof-dir`][]
 * [`--redirect-warnings`][]
 
+### `--disable-warnings=code-or-type`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Disable specific process warnings by `code` or `type`.
+
+Warnings emitted from [`process.emitWarning()`][emit_warning] may contain a
+`code` and a `type`. This option will not-emit warnings that have a matching
+`code` or `type`.
+
+List of [deprecation warnings][].
+
+The Node.js core warning types are: `DeprecationWarning` and
+`ExperimentalWarning`
+
+For example, the following script will not emit
+[DEP0025 `require('node:sys')`][DEP0025 warning] when executed with
+`node --disable-warnings=DEP0025`:
+
+<!-- eslint-skip -->
```

Comment:
Can you specify the rule that's being skipped? So if another lint error appears, we can spot it.

Reply:
<img width="347" alt="Screenshot 2023-11-20 at 11 31 35" src="https://github.com/nodejs/node/assets/16144158/62a4974e-e275-438d-b8c0-3f11b9d022c2">

I'm not sure how to do this. No other `eslint-skip` comments seem to do so either.

Reply:
Maybe just mention in a comment?

Reply:
I've opened https://github.com/nodejs/node/pull/50829 to address that. You can use that as an example of how to do that.

Reply:
Interestingly enough, removing the comments and running the linter again results in no issue. Not sure why it was throwing before and not now

Reply:
Maybe the CI will throw something 🤷‍♂️